pepper-apis/src/test/java/org/broadinstitute/ddp/route/IntegrationTestSuite.java

```diff
@@ -336,4 +361,30 @@ public static void doSuiteTearDown() {
             IntegrationTestSuite.tearDown();
         }
     }
+
+    private static boolean isTestServerRunning() {
+        int serverPortNum = ConfigManager.getInstance().getConfig().getInt(ConfigFile.PORT);
+        String serverUrl = "http://localhost:" + serverPortNum;
+        try (
+                CloseableHttpClient httpClient = HttpClients.createDefault();
+
+        ) {
+            httpClient.execute(new HttpGet(serverUrl));
+            return true;
+        } catch (HttpHostConnectException e) {
+            LOG.debug("Could not connect to server url. Must not be running");
+            return false;
+        } catch (IOException e) {
+            String msg = "There was problem initializing CloseableHttpClient";
+            LOG.error(msg, e);
+            throw new DDPException(msg, e);
+        }
+
+    }
+
+    public static void main(String[] args) {
+        initializeDatabase();
+        startupTestServer();
+    }
+
```

Comment:
Refactored IntegrationSuite here so we can start a test server that has applied to the common database changes. Tried to preserve existing functionality. Definitely room for improvement here.